bindings/c/include/cgenalyzer_private.h

```diff
@@ -60,172 +42,184 @@ constexpr int gn_failure = 1;
 
 namespace gn = ::genalyzer_impl;
 
-namespace util {
-
-    class log
-    {
-    public:
-
-        log()
-            : m_log {},
-              m_flag {false}
-        {}
-
-    public:
-
-        bool check()
-        {
-            return m_flag;
-        }
-
-        void clear()
-        {
-            m_log.clear();
-            m_flag = false;
-        }
-    
-        std::string_view get() const
-        {
-            return m_log;
-        }
-    
-        void set(const char* msg)
-        {
-            m_log = msg;
-            m_flag = true;
-        }
-    
-        size_t size() const
-        {
-            return m_log.size();
-        }
-    
-    public:
-
-        void append() {}
-
-        template<typename... Types>
-        void append(const char* s, Types... the_rest)
-        {
-            m_log.append(s);
-            append(the_rest...);
-        }
-    
-        void prepend() {}
-    
-        template<typename... Types>
-        void prepend(const char* s, Types... the_rest)
-        {
-            prepend(the_rest...);
-            m_log.insert(0, s);
-        }
-    
-    private:
-    
-        std::string m_log;
-        bool m_flag;
-
-    }; // class log
-
-    static log gn_error_log;
-
-    template<typename... Types>
-    int return_on_exception(const char* s, Types... the_rest)
-    {
-        gn_error_log.set(s);
-        gn_error_log.append(the_rest...);
-        return gn_failure;
-    }
+namespace util
+{
+
+class log
+{
+public:
+  log () : m_log{}, m_flag{ false } {}
+
+public:
+  bool
```

Comment:
The return types all inconsistent between files. I would say they look better on the same line as the function definition and not on a separate line

Reply:
Updated.